src/encoded/static/components/audit.js

```diff
@@ -1,168 +1,350 @@
+        const categoryName = group[0].category.uppercaseFirstChar();
+
+        return (
+            <div className={alertClass}>
+                <div className={`icon audit-detail-trigger-${level}`}>
+                    <button onClick={this.detailSwitch} className="collapsing-title">
+                        {collapseIcon(!detailOpen)}
                     </button>
-                );
-            }
+                </div>
+                <div className="audit-detail-info">
+                    <i className={iconClass} />
+                    <strong>&nbsp;{categoryName}</strong>
+                    <div className="btn-info-audit">
+                        <a href={`/data-standards/audits/#${categoryName.toLowerCase().split(' ').join('_')}`} title={`View description of ${categoryName} in a new tab`} rel="noopener noreferrer" target="_blank"><i className="icon icon-question-circle" /></a>
+                    </div>
+                </div>
+                <div className="audit-details-section">
+                    <div className="audit-details-decoration" />
+                    {group.map(audit =>
+                        <div className={alertItemClass} key={audit.id} role="alert">
+                            <DetailEmbeddedLink detail={audit.detail} except={except} forcedEditLink={forcedEditLink} />
+                        </div>,
+                    )}
+                </div>
+            </div>
+        );
+    }
+}
 
-            // Logged out and the only audit level is DCC action, so don't show a button
-            return null;
-        } else {
-            return null;
-        }
+AuditGroup.propTypes = {
+    group: PropTypes.array.isRequired, // Array of audits in one name/category
+    auditLevelName: PropTypes.string.isRequired, // Audit level name
+    except: PropTypes.string, // @id of object whose audits are being displayed.
+    forcedEditLink: PropTypes.bool, // `true` to display the `except` path as a link anyway
+};
+
+AuditGroup.defaultProps = {
+    except: '',
+    forcedEditLink: false,
+};
+
+
+// No necessarily distinguishing information for each audit, making it impossible to reliably key
+// them as React components. This takes an audit object as it arrives in a parent data object and
+// adds a unique id to each one that we can use as a key when rendering each one. Uses an npm
+// module that generates guaranteed unique values.
+function idAudits(audits) {
+    Object.keys(audits).forEach((auditType) => {
+        const typeAudits = audits[auditType];
+        typeAudits.forEach((audit) => {
+            audit.id = shortid.generate();
+        });
+    });
+}
+
+
+// Determine whether audits should be displayed or not, given the audits object from a data object,
+// and an optional session to determine logged-in state. It follows this logic:
+//
+// if audits exist
+//     if logged out
+//         if only one audit level and it's INTERNAL_ACTION
+//             return do display audits
```

Comment:
For [ENCSR878KIY](https://www.encodeproject.org/experiments/ENCSR878KIY/) I see “Biological replicates with identical biosample” which is an INTERNAL_ACTION only visible when you’re logged in.